crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/storage.rs

```diff
@@ -13,26 +13,24 @@ use starknet_api::state::StorageKey;
 ///
 /// * `blockifier_state`: Blockifier state reader
 /// * `target`: The address of the contract we want to target
-/// * `storage_address`: Beginning of the storage of the variable
-/// * `values`: A vector of values to write starting at `storage_address`
+/// * `storage_address`: Storage address of the felt
+/// * `value`: A felt value to write at `storage_address`
 ///
 /// returns: Result<(), Error> - a result containing the error if `store` failed  
 ///
 pub fn store(
     blockifier_state: &mut BlockifierState,
     target: ContractAddress,
     storage_address: &Felt252,
-    values: &[Felt252],
+    value: Felt252,
 ) -> Result<(), anyhow::Error> {
-    for (i, value) in values.iter().enumerate() {
-        blockifier_state.blockifier_state.set_storage_at(
-            target,
-            StorageKey(PatriciaKey::try_from(StarkHash::from_(
-                storage_address.clone() + Felt252::from(i),
-            ))?),
-            value.clone().into_(),
-        );
-    }
+    blockifier_state.blockifier_state.set_storage_at(
+        target,
+        StorageKey(PatriciaKey::try_from(StarkHash::from_(
+            storage_address.clone(),
+        ))?),
```

Comment:
Let's make this into a function

Reply:
[Pack making storage key into a function](https://github.com/foundry-rs/starknet-foundry/pull/1487/commits/fa6ea536aa6e24ec182c4ce1b7ad6dc78d684e96)